app/Http/Controllers/Api/CommentController.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace App\Http\Controllers\Api;
+
+use App\Comment;
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Services\CommentService;
+use App\Http\Resources\Comment as CommentResource;
+use App\Http\Requests\CreateCommentRequest;
+use App\Http\Requests\CheckMovieExistPathRequest;
+
+class CommentController extends Controller
+{
+    private $commentService;
+
+    public function __construct(CommentService $commentService)
+    {
+       $this->commentService = $commentService;
+    }
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $collection = $this->commentService->findAll();
+        $result =[];
+        foreach($collection as $c)
```

Comment:
ovo mi se cini da bi moglo prostije, imamo multi i single resource

Reply:
Pogledacu to :)